lib/rubocop/cop/minitest/refute_includes.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Check if your test uses `refute_includes`
```

Comment:
I think this is worded confusingly as now it implies that `refutes_includes` is what's bad.